packages/gestalt/src/behaviors/TrapFocusBehavior.js

```diff
@@ -53,7 +53,8 @@ export default function TrapFocusBehavior({ children }: Props): ReactNode {
 
     // Focus the first child element among all the focusable, interactive elements within `children`
     const focusFirstChild = () => {
-      if (element) {
+      const withinIframe = window !== window.parent;
```

Comment:
Should we add a comment about why we're checking for iframes? This doesn't seem self-explanatory outside of the context of our docs pages